### ISSUE TITLE
No exported member ɵBrowserPlatformLocation in single-spa-angular/src/extra-providers.d.ts

### ISSUE THREAD

**Issue by r0boto** [single-spa/single-spa-angular#477]
### Demonstration

Trying to run the app using:
`"serve:single-spa:angular-app1": "ng s --project angular-app1 --disable-host-check --port 4200 --live-reload false"`

And Im getting the following:

```
Error: node_modules/single-spa-angular/src/extra-providers.d.ts:2:10 - error TS2724: '"@angular/common"' has no exported member named 'ɵBrowserPlatformLocation'. Did you mean 'BrowserPlatformLocation'?

2 import { ɵBrowserPlatformLocation, LocationChangeListener } from '@angular/common';
           ~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@angular/common/index.d.ts:113:22
    113 export declare class BrowserPlatformLocation extends PlatformLocation {
                             ~~~~~~~~~~~~~~~~~~~~~~~
    'BrowserPlatformLocation' is declared here.
```
    
    
    dependencies versions from the package.json 
    
   ```
   "dependencies": {
    "@angular/animations": "^15.2.0",
    "@angular/common": "^15.2.0",
    "@angular/compiler": "^15.2.0",
    "@angular/core": "^15.2.0",
    "@angular/forms": "^15.2.0",
    "@angular/platform-browser": "^15.2.0",
    "@angular/platform-browser-dynamic": "^15.2.0",
    "@angular/router": "^15.2.0",
    "rxjs": "~7.8.0",
    "single-spa": ">=4.0.0",
    "single-spa-angular": "^8.0.1",
    "tslib": "^2.3.0",
    "zone.js": "~0.12.0"
  },
  "devDependencies": {
    "@angular-builders/custom-webpack": "15.0.0",
    "@angular-devkit/build-angular": "^15.2.1",
    "@angular/cli": "~15.2.1",
    "@angular/compiler-cli": "^15.2.0",
    "@types/jasmine": "~4.3.0",
    "jasmine-core": "~4.5.0",
    "karma": "~6.4.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.2.0",
    "karma-jasmine": "~5.1.0",
    "karma-jasmine-html-reporter": "~2.0.0",
    "style-loader": "^3.3.1",
    "typescript": "~4.9.4"
  }
```

**ng version**          

Angular CLI: 15.2.2
Node: 18.15.0
Package Manager: npm 9.5.0
OS: darwin arm64

Angular: 15.2.2
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1502.2
@angular-devkit/build-angular   15.2.2
@angular-devkit/core            15.2.2
@angular-devkit/schematics      15.2.2
@schematics/angular             15.2.2
rxjs                            7.8.0
typescript                      4.9.5

Single spa with version "@angular/common": "^15.1.0" ae working correctly.

### Expected Behavior
Updated single-spa-angular/src/extra-providers file
No exception on Angular 15.2.2

### Actual Behavior
The exception is thrown.

**Comment by zachschroeder** [comment]
My team is also experiencing this.

**Comment by DDtMM** [comment]
Yep.. Only solution was to downgrade to 15.2.1.  This has been deprecated for a long time.

**Comment by AdibW** [comment]
@DDtMM I have already reverted to a previous version of my application, effectively downgrading it from 15.2.2 to 15.2.1, but I still have the same issues.

**Comment by zachschroeder** [comment]
The solution that my team and I have used is to downgrade all @angular packages to 15.2.1. After doing this, we no longer saw this error.

**Comment by arturovt** [comment]
Published `8.1.0`. The minimal required version of Angular is 15.1.0. Feel free to re-open once any follow-up issues are faced.
